src/package-install-scripts.js

```diff
@@ -3,12 +3,17 @@
 import type {Manifest} from './types.js';
 import type PackageResolver from './package-resolver.js';
 import type {Reporter} from './reporters/index.js';
-import type Config from './config.js';
+import Config from './config.js';
 import type {ReporterSetSpinner} from './reporters/types.js';
 import executeLifecycleScript from './util/execute-lifecycle-script.js';
+import * as crypto from './util/crypto.js';
 import * as fs from './util/fs.js';
+import {getPlatformSpecificPackageFilename} from './util/package-name-utils.js';
+import {pack} from './cli/commands/pack.js';
 
+const fs2 = require('fs');
```

Comment:
I find that a bit confusing - maybe createWriteStream should be simply reexported from `util/fs` (or maybe we should use `fs-extra` which takes care of exporting promises when we need them).

Reply:
Yeah, fs2 is a poor choice for naming this variable.
Let's do it like in other places the Node.js fs module should be `fs` but out promise wrappers should be named `fsUtil`.

A year ago there was a proposal to reexport all exports from Node.js `fs` module there but we decided not to do it because it means we would have to manually list all functions of Node.js `fs`. Could be just extra trouble to worry when Node.js introduces a new function or breaks API

Reply:
`fs-extra` automatically exports all the `fs` Node functions in such a way that they automatically support promises. Using it would allow us to let someone else care about keeping the list updated! :) Ok, not urgent, let's keep this for later